test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -10,73 +10,70 @@
 
 
 SMDEBUG_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3.8xlarge", processor="gpu")
-SMDEBUG_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c5.9xlarge", processor="cpu")
+SMDEBUG_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c4.8xlarge", processor="cpu")
```

Comment:
Changed instance type to reflect a commonly tested cpu instance type in PR context. Left p3.8x as is so that this is exercised on p3 instance types in PR context. Can change back if folks have strong opinions about this.